addons/io/org.openhab.io.hueemulation/README.md

```diff
@@ -10,7 +10,7 @@ Hue Emulation exposes openHAB items as Hue devices to other Hue HTTP API compati
 * Pairing (security) can be enabled/disabled in real time using the configuration service (under services in the PaperUI for example)  
 
 ## Configuration:
-
+in "userdata/config/org/openhab/hueemulation.config"
```

Comment:
Right, thanks for that. But the docs should nonetheless contain proper sentences.
And afaik, the correct place for such a configuration is conf/services/runtime.cfg - I guess this should be covered in general by the main documentation through https://github.com/openhab/openhab-docs/issues/309.